platform/osx/os_osx.mm

```diff
@@ -2909,11 +2909,10 @@ static int get_screen_index(NSScreen *screen) {
 			layered_window = false;
 		}
 		[context update];
-		NSRect frame = [window_object frame];
-
 		if (!is_no_window_mode_enabled()) {
-			[window_object setFrame:NSMakeRect(frame.origin.x, frame.origin.y, 1, 1) display:YES];
-			[window_object setFrame:frame display:YES];
+			NSRect frameRect = [window_object frame];
+			[window_object setFrame:NSMakeRect(frameRect.origin.x, frameRect.origin.y, frameRect.size.width + 1, frameRect.size.height) display:NO];
```

Comment:
Might be worth adding a comment to clarify the purpose of this call (especially with `width + 1`, which I guess is to enforce a redraw with a different size?).

Reply:
Added comment, yes it's resizing window a bit to trigger GL context update.